modApiServer/src/org/aion/api/server/ApiAion.java

```diff
@@ -50,6 +50,7 @@
 import org.aion.zero.types.AionTxReceipt;
 
 public abstract class ApiAion extends Api {
+    public static final int SYNC_TOLERANCE = 5;
```

Comment:
Is there any reason not to just make this 1?

Reply:
I guess realistically you can be within ~50 blocks and you will be safe to send a transaction to the node because by the time you do you will be at the top, so I don't think there's much of a difference for any value below this point, and especially for any values below 10.

Since the old value was 5 I don't think there's much point in changing it really? But if you guys want to see this change to 1 that's probably fine.. though we should run the test harness against that value just to ensure there's no weirdness from how close this value is to zero. Just for this extra work I feel we should leave it be since there's no practical difference.

Reply:
It's not a huge deal (especially considering this function did the opposite of what it was supposed to for so long)... it's just -- when it's 1, it's easier for the user to figure out network best block. Because if you eth_syncing is returning false, you know that networkBest=localBest, so you can eth_getBlockNumber and know what localBest and networkBest are.  Whereas if it's 5, then when it returns false, the you're only able to deduce that networkBest is in [localBest-5,localBest].

Reply:
Fair enough. I will hopefully get around to testing this out against the harness soon just to see that it behaves okay and that 1 isn't an issue. I don't suspect it will be.